pages/publishing/publishing/adobe-experience-platform.mdx

```diff
@@ -0,0 +1,81 @@
+import {Callout} from 'nextra/components'
+
+# Adobe Experience Platform
+
+*Publishing your tracking plan to the Adobe Experience Platform*
+
+![Adobe Logo](/images/publishing_adobe.svg)
+
+## Introduction
+
+The integration to [Adobe Experience Platform XDM system](https://experienceleague.adobe.com/en/docs/experience-platform/xdm/home) allows you to publish your Avo Tracking Plan as XDM Schemas to Adobe Experience Platform in form of XDM schemas. You can set up the integration to automatically publish on every Avo branch merge or manually trigger a publish. 
+
+## Configuring the Adobe Experience Platform integration
+
+To enable the direct integration with AEP, navigate to the Publishing Screen in your Avo workspace, click “Add Integration” and select “Adobe Experience Platform. 
```

Comment:
Shoukld we link to this here? https://www.avo.app/docs/publishing/publishing/overview#setting-up-and-configuring-a-publishing-integration. 

Maybe just end this paragraph with "see details in our setting up and configuring a publishing integration dcs"